tests/test_download.py

```diff
@@ -105,96 +109,28 @@ def html_string():
             </section>
         </body>
     </html>"""
-    return html
 
 
 class TestDownload:
-    def test_imports(self):
-        from nemo_curator.download import (
-            JusTextExtractor,
-            ResiliparseExtractor,
-            TrafilaturaExtractor,
-            download_arxiv,
-            download_common_crawl,
-            download_wikipedia,
-        )
-
+    def test_imports(self) -> None:
```

Comment:
Ruff seems to think they're not needed. But that's what the test is. Let's add these back along with noqa so that they're not removed in a subsequent `ruff check --fix`